.gitignore

```diff
@@ -21,3 +21,15 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+# VisualStudioCode #
+.vscode/*
+!.vscode/settings.json
+!.vscode/tasks.json
+!.vscode/launch.json
+!.vscode/extensions.json
```

Comment:
Not sure what these `!` mean, are they excluded or included?

Reply:
It's included. I grabbed it from template though. I probably should remove them.